$NetBSD: patch-build_moz.configure_rust.configure,v 1.4 2018/09/14 14:54:19 ryoon Exp $

* fuchsia OS "triplet" is notb triplet. Convert the "triplet" to triplet.

--- build/moz.configure/rust.configure.orig	2018-08-30 16:44:17.000000000 +0000
+++ build/moz.configure/rust.configure
@@ -119,6 +119,8 @@ def rust_supported_targets(rustc):
     ambiguous = set()
     per_raw_os = {}
     for t in out:
+        if t == 'aarch64-fuchsia' or t == 'x86_64-fuchsia':
+            t = t.replace('-', '--')
         t = split_triplet(t, allow_unknown=True)
         key = (t.cpu, t.endianness, t.os)
         if key in per_os:
